compiler/qsc_qasm3/src/compile/tests.rs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+use crate::tests::{parse_all, print_compilation_errors, qasm_to_program_fragments};
+use miette::Report;
+
+#[test]
+fn programs_with_includes_with_includes_can_be_parsed() -> miette::Result<(), Vec<Report>> {
```

Comment:
I think this may be a copy-paste thing, but this test case has the same name as the third one in parse/tests.rs, which might cause minor confusion when looking at test output (or doing a find all). Could this one be renamed to be compile specific rather than use "parsed?"

```suggestion
fn programs_with_includes_with_includes_can_be_compiled() -> miette::Result<(), Vec<Report>> {
```